game/4x4_evolution_pod.ksy

```diff
@@ -0,0 +1,40 @@
+meta:
+  id: game_4x4_evolution_pod
+  file-extension: pod
+  application: 4x4 Evolution
+  license: CC0-1.0
+  endian: le
+  encoding: utf-8
+doc-ref: 'https://www.watto.org/specs.html?specs=Archive_POD'
+seq:
+  - id: header
+    type: archive_header
+  - id: details_directory
+    type: details_directory
```

Comment:
The type should not be called `details_directory`, because that's not what the type represents. It represents a single file, so naming it a "directory" is confusing. Using `details_directory` as the name of the array is somewhat less confusing, but it doesn't comply with the naming convention specified [in the style guide](https://doc.kaitai.io/ksy_style_guide.html#attr-id):

> - For an array of objects (i.e. with `repeat: something`), use plural form — e.g. `files`, `transactions`.

So I'd personally go for `files` and the type `file`.